### PR TITLE
Adjusted margin of first rail image

### LINKED ISSUE #244: Article - rail image
Top align first rail image with key line under author name

### PR DESCRIPTION
**Pull request by gestchild**
## What is this PR trying to achieve?
Fixes #244

## What does it look like?

Before: 
screen shot 2017-02-08 at 15 48 15

After (with author):
screen shot 2017-02-08 at 16 00 24

After (without author):
screen shot 2017-02-08 at 16 00 46

### CODE REVIEW

**Comment by Norvard** [comment]
Looking sweet!

File: client/scss/components/_body_content.scss [hunk 1]
```diff
@@ -94,6 +94,10 @@ $center-width-xlarge: (6/6) * 100%;
       clear: right;
       width: #{$narrow-width};
       margin-right: calc((#{$narrow-width} + #{map-get($gutter-width, 'large')} ) * -1);
+
+      .author + .article &:first-child {
```
Comment (jamesgorrie) [inline client/scss/components/_body_content.scss]: Would adding a class to this and target that be safer?
e.g. if the first child is the paragraph - do we still want to do this?

Comment (gestchild) [inline client/scss/components/_body_content.scss]: The & refers to the .body-part--sticky class, so won't apply to paragraphs

Comment (jamesgorrie) [inline client/scss/components/_body_content.scss]: Bah! I wasn't reading the diff correctly. 👍

**Pull request merged**